images/miq-app/ose-assets/appliance-initialize.service

```diff
@@ -1,13 +1,13 @@
 [Unit]
 Description=Initialize Appliance Database
-After=evminit.service memcached.service
+After=evminit.service
 Before=evmserverd.service
-Requires=memcached.service
 [Service]
 TimeoutStartSec=10m
 Type=oneshot
+EnvironmentFile=/container.env.vars
```

Comment:
@fvillain I don't think is needed anymore, in addition this approach is more elegant and friendly with systemd. When you finish I go testing again and we will see what happens :smiley: